script.js

```diff
@@ -1,4 +1,123 @@
+// Global Variables
+// Track player 1 score as integer
+var intPlayerOneScore = 0;
+var intPlayerOneScoreAlt = 0;
+// Track player 1 score as string
+var playerOneScoreString = 0;
+var playerOneScoreAltString = 0;
+// Track player 2 score as integer
+var intPlayerTwoScore = 0;
+var intPlayerTwoScoreAlt = 0;
+// Track player 2 score as string
+var playerTwoScoreString = 0;
+var playerTwoScoreStringAlt = 0;
+// Tracking Final Score of Both Players
+var finalScorePlayerOne = "";
+var finalScorePlayerTwo = "";
+// gameTurns
+var gameTurn = "Player 1";
+
+//Create a diceRoll function
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger;
+};
+
+// Score comparison function
+var scoreComparison;
+
```

Comment:
Hey bro, without going into the individual lines, I thought your game's UX is pretty refreshing because you thought of a slightly different way (A and B) to make visualizing the order of the numbers a bit more intuitive.

However, I encourage you to implement some data validation. e.g. If I entered "A" instead of "a", I would get an "undefined" which is not ideal.

I also encourage you to implement a game reset functionality instead of letting the game end after 1 round